pull.go

```diff
@@ -54,6 +55,11 @@ func (pull *pullSocket) Type() SocketType {
 	return pull.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.